libs/dh/shared/feature-authorization/src/lib/permission.service.ts

```diff
@@ -36,24 +41,32 @@ export class PermissionService {
       return of(true);
     }
 
-    const account = this.authService.instance.getActiveAccount();
+    const accounts = this.authService.instance.getAllAccounts();
+
+    // we expect one and only one account for now.
+    if (accounts.length != 1) {
+      return of(false);
+    }
+
+    const account = accounts[0];
 
     // If MSAL returns no accounts or the claims are missing,
     // the service default to no access.
     if (!account) {
```

Comment:
issue: this `if` statement will pretty much always be `true` because an object (either empty or with properties) is truthy. With that in mind, is the logic stil relevant?

Reply:
You're right. This is leftover from when we used getActiveAccount. Fixed.